<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureGeneral</class>
 <widget class="QWidget" name="ConfigureGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>General</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QCheckBox" name="toggle_check_exit">
          <property name="text">
           <string>Confirm exit while emulation is running</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="toggle_background_pause">
          <property name="text">
           <string>Pause emulation when in background</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="toggle_hide_mouse">
          <property name="text">
           <string>Hide mouse on inactivity</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="updateBox">
       <property name="title">
        <string>Updates</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCheckBox" name="toggle_update_check">
          <property name="text">
           <string>Check for updates on start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="toggle_auto_update">
          <property name="text">
           <string>Silently auto update after closing</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Emulation</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Region:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="region_combobox">
          <item>
           <property name="text">
            <string>Auto-select</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">JPN</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">USA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">EUR</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">AUS</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">CHN</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">KOR</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">TWN</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="toggle_frame_limit">
          <property name="text">
           <string>Limit Speed Percent</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="frame_limit">
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="button_reset_defaults">
       <property name="text">
        <string>Reset All Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
